modules/SecurityGroups/metadata/detailviewdefs.php

```diff
@@ -2,45 +2,44 @@
 
 $module_name = 'SecurityGroups';
 $viewdefs[$module_name]['DetailView'] = array(
-'templateMeta' => array('form' => array('buttons'=>array('EDIT', 'DUPLICATE', 'DELETE',
-                                                         )),
-                        'maxColumns' => '2',
-                        'widths' => array(
-                                        array('label' => '10', 'field' => '30'),
-                                        array('label' => '10', 'field' => '30')
-                                        ),
-                        ),
-                        
-'panels' =>array(
-  
-  array(
-    'name',
-    'assigned_user_name',
-  ),
-
-
-
+    'templateMeta' => array('form' => array('buttons' => array('EDIT', 'DUPLICATE', 'DELETE',
```

Comment:
Hi @jack7anderson7, since we are taking the chance to re-format. Could we fix the formatting for this line also, please?